src/content/overview/observability/configuration/_index.md

```diff
@@ -0,0 +1,170 @@
+
+## Data ingestion configuration
+
+Control what observability data flows into your platform and how it's processed:
+
+### Metrics collection
+
+Configure **[metrics ingestion]({{< relref "/tutorials/observability/data-ingestion/metrics" >}})** to collect custom application and infrastructure metrics:
+
+- **ServiceMonitors**: Define which services to scrape for metrics and how frequently
+- **PodMonitors**: Configure direct pod-level metric collection for specialized use cases
+- **Tenant routing**: Specify which tenant receives metrics through labeling
+- **Metric transformation**: Apply relabeling rules to modify, filter, or enrich metrics before storage
+
+### Log collection
+
+Configure **[log ingestion]({{< relref "/tutorials/observability/data-ingestion/logs" >}})** to capture application and system logs:
+
+- **PodLogs**: Select which pods to collect logs from using label selectors
+- **Pod labels**: Enable automatic log collection by labeling pods with tenant information
+- **Log parsing**: Configure structured parsing and label extraction from log content
+- **Namespace filtering**: Control log collection scope by namespace
+
+### External data sources
+
+Integrate data from sources outside Giant Swarm managed clusters through the **[Observability Platform API]({{< relref "/overview/observability/observability-platform-api" >}})**:
+
+- **OIDC authentication**: Configure your identity provider for secure API access
+- **Data ingestion endpoints**: Send metrics, logs, and traces from external systems
+- **OpenTelemetry Protocol (OTLP)**: Configure standardized telemetry data ingestion
+
+## Visualization and dashboard configuration
+
+Customize how you view and visualize your observability data:
+
+### Grafana organizations
+
+Once you've created Grafana organizations as part of your multi-tenancy setup, you can leverage them for powerful visualization capabilities:
+
+- **Isolated workspaces**: Each organization provides a dedicated environment for team-specific dashboards and visualizations
+- **Tenant-scoped data access**: Automatically provisioned datasources provide teams with access only to their authorized data
+- **Collaborative visualization**: Share dashboards and insights within organization boundaries
+- **Custom dashboard organization**: Organize monitoring assets by team, environment, or service ownership
+
+### Custom dashboards
+
+**[Create custom dashboards]({{< relref "/tutorials/observability/data-exploration/creating-custom-dashboards" >}})** tailored to your specific monitoring needs:
+
+- **GitOps approach**: Version-controlled dashboard definitions using ConfigMaps
+- **UI creation**: Build dashboards directly in Grafana with persistent storage
+- **Dashboard provisioning**: Automatically deploy dashboards to specific organizations
```

Comment:
Which only works in the GitOps Approach, right? So I would put it to the order: 
- GitOps approach
- Dashboard provisioning
- UI creation
- Cross-tenant dashboards